jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -15,6 +15,7 @@
 package io.jaegertracing;
 
 import io.jaegertracing.internal.JaegerTracer;
+import io.jaegertracing.internal.JaegerObjectFactory;
```

Comment:
shouldn't need this

Reply:
Ya I use the Google formatting plugin that will adjust the import and formatting at once, so I held off to avoid the formatting issues from earlier.